6_visualize/src/prep_legend_fun.R

```diff
@@ -67,32 +72,45 @@ prep_legend_fun <- function(precip_bins, legend_styles, timesteps_ind, storm_poi
-    points(dot_x, normal_y+center_to_txt_y, pch = 21, bg = legend_styles$gage_norm_col, col = NA, cex = 2)
-    text(dot_txt_x, normal_y, labels = 'Below flood stage', pos = txt_pos,
+    text(dot_txt_x_gage+bin_w*1.75, normal_y, labels = 'Below', pos = txt_pos,
          cex=legend_text_cfg$cex, col=legend_text_cfg$col, family=legend_text_cfg$family)
-    flood_y <- ybottom+3*bin_h
-    points(dot_x, flood_y+center_to_txt_y, pch = 21, bg = legend_styles$gage_norm_col, col = legend_styles$gage_flood_col, lwd = 4, cex = 2)
-    text(dot_txt_x, flood_y, labels = 'Above flood stage', pos = txt_pos,
+    flood_y <- coord_space[4]*0.9
+    points(dot_x_gage+bin_w*3.5, flood_y+center_to_txt_y, pch = 21, bg = legend_styles$gage_norm_col, col = legend_styles$gage_flood_col, lwd = 4, cex = 2)
+    text(dot_txt_x_gage+bin_w*3.5, flood_y, labels = 'Above', pos = txt_pos,
          cex=legend_text_cfg$cex, col=legend_text_cfg$col, family=legend_text_cfg$family)
+    }
 
     # plot storm legend
-    hurricane_y <- ybottom+6*bin_h*1.05
-    if(has_storm_track) {
-      if(is.na(hurricane_col)) {
-        # When the hurricane dot is no longer visible, switch to showing just the path
-        text(dot_txt_x, hurricane_y, labels = paste("Path of", legend_styles$storm_name), pos = txt_pos,
-             cex=legend_text_cfg$cex, col=legend_text_cfg$col, family=legend_text_cfg$family)
-        segments(seg_x, hurricane_y+center_to_txt_y, dot_txt_x, lty = "dotted",
-                 col = legend_styles$storm_line_col, lwd = 2)
-      } else {
-        text(dot_txt_x, hurricane_y, labels = sprintf(hurricane_cat, legend_styles$storm_name), pos = txt_pos,
-             cex=legend_text_cfg$cex, col=legend_text_cfg$col, family=legend_text_cfg$family)
-        points(dot_x, hurricane_y+center_to_txt_y, pch = 21, bg = hurricane_col, col = NA, cex = 3)
+    hurricane_y <- coord_space[3]*1.3
+
+
+    if(intro == FALSE){
+      if(has_storm_track) {
+        if(is.na(hurricane_col)) {
+          # When the hurricane dot is no longer visible, switch to showing just the path
+          text(dot_txt_x, hurricane_y, labels = paste("Path of", legend_styles$storm_name), pos = txt_pos,
+               cex=legend_text_cfg$cex, col=legend_text_cfg$col, family=legend_text_cfg$family)
+
+          segments(seg_x, hurricane_y+center_to_txt_y, dot_txt_x, lty = "dotted",
+                   col = legend_styles$storm_line_col, lwd = 2)
+        } else {
+          text(dot_txt_x, hurricane_y, labels = sprintf(hurricane_cat, legend_styles$storm_name), pos = txt_pos,
+               cex=legend_text_cfg$cex, col=legend_text_cfg$col, family=legend_text_cfg$family)
+          text(dot_txt_x, hurricane_y+coord_space[4]*0.95, labels = legend_styles$storm_name, pos = txt_pos,
+               cex=legend_text_cfg$cex, col=legend_text_cfg$col, family=legend_text_cfg$family)
+          points(dot_x, hurricane_y+center_to_txt_y, pch = 21, bg = hurricane_col, col = NA, cex = 3)
+        }
       }
+
```

Comment:
Do you want to commit your adjustments to the legend to the `main` branch? I really liked the adjustments